api/routes/zetter.js

```diff
@@ -3,12 +3,14 @@ const router = express.Router();
 const { verify } = require('jsonwebtoken');
 const users = require('../data/users.js');
 
-const { getTweets, createTweet, login } = require('../controllers/zetter.js');
+const { getTweets, createTweet, getProfile, updateProfile, login } = require('../controllers/zetter.js');
 
 router.get('/', auth, getTweets);
-
 router.post('/', auth, createTweet);
 
+router.get('/profile', auth, getProfile);
+router.put('/update/profile', auth, updateProfile);
```

Comment:
```suggestion
router.put('/profile', auth, updateProfile);
```
または
```suggestion
router.patch('/profile', auth, updateProfile);
```
とする

RESTfulとよばれるAPIエンドポイントの設計思想では、

- URLは「リソース」を表現し、できるだけ「操作」を表現しないようにする
- 「操作」は、メソッドやヘッダーを使って表現する

というのが原則で、今回でいうと「profileを(=リソース)」「updateしたい(=操作)」の操作の部分は`PUT`または`PATCH`メソッドを使うことで表現する

Reply:
参考
https://qiita.com/NagaokaKenichi/items/6298eb8960570c7ad2e9